app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -286,9 +288,13 @@ class EventDetailsFragment : Fragment() {
         rootView.refundPolicy.text = event.refundPolicy
 
         // Similar Events Section
-        if (event.eventTopic != null) {
+        if (event.eventTopic != null || !event.locationName.isNullOrBlank() ||
+            !event.searchableLocationName.isNullOrBlank()) {
             similarEventsContainer.visibility = View.VISIBLE
```

Comment:
Not related to your PR, but we could use `view.isVisible=true` for following kotlin coding style. Concise and cleaner meaning

Reply:
good that's for pointing that out will change

Reply:
resolved